pages/JuiceShop/ProductsPage.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const BasePage = require('../basePage.js');
+const LoginPage = require('./LoginPage.js');
+
+class ProductsPage extends BasePage {
+
+    constructor() {
+        super();
+    }
+
+    get searchIcon() { return browser.$('.mat-search_icon-search'); }
+    get searchInput() { return browser.$('#mat-input-0'); }
+    get searchedItem() { return browser.$$('.product').then(el => el[0]); }
+    get noResultTitle() { return browser.$$('.noResultText').then(el => el[0]); }
+    get noResultMessage() { return browser.$$('.noResultText').then(el => el[1]); }
+    get clearSearchIcon() { return browser.$('.mat-search_icon-close'); }
+
+    async navigateToProductsPage() {
+        await this.navigateTo('https://juice-shop.herokuapp.com/#/');
+        return this.waitForElement(LoginPage.welcomeDialog);
+    }
+
+    async searchForProduct(itemName) {
+        await this.clickOnElement(this.searchIcon);
+        await this.changeElementText(this.searchInput, itemName);
+        await browser.keys("\uE007");

```

Comment:
Please avoid usage of this approach in future because it is hard to understand what key you send to the browser. In the similar situations please use `addValue` or `setValue` methods. By default, if you pass normalized key value from [this list](https://w3c.github.io/webdriver/#keyboard-actions) to one of these methods, wdio will automatically transform it to the unicode. For example,
```
const inputField = await $('input[name=\'age\']');
await inputField.addValue('Enter');
```